File: workspace-servidor/flotix2021/src/main/java/com/flotix/response/bean/ServerResponseAlquiler.java
package com.flotix.response.bean;

import java.util.List;

import com.flotix.dto.AlquilerDTO;

/**
 * Objeto de respuesta con la informacion del alquiler
 * 
 * @author Flor
 *
 */
public class ServerResponseAlquiler {

	private String idAlquiler = null;

	private List<AlquilerDTO> listaAlquiler = null;

	private ErrorBean error = new ErrorBean();

	public ServerResponseAlquiler() {
		super();
	}

	public String getIdAlquiler() {
		return idAlquiler;
	}

	public void setIdAlquiler(String idAlquiler) {
		this.idAlquiler = idAlquiler;
	}

	public List<AlquilerDTO> getListaAlquiler() {
		return listaAlquiler;
	}

	public void setListaAlquiler(List<AlquilerDTO> listaAlquiler) {
		this.listaAlquiler = listaAlquiler;
	}

	public ErrorBean getError() {
		return error;
	}

	public void setError(ErrorBean error) {
		this.error = error;
	}
}

File: workspace_presentacion/Flotix2021/Flotix2021/ModelResponse/ServerResponseAlquiler.cs
using Flotix2021.ModelDTO;
using System;
using System.Collections.Generic;

namespace Flotix2021.ModelResponse
{
    class ServerResponseAlquiler
    {
        private string _idAlquiler;

        public string idAlquiler
        {
            get { return _idAlquiler; }
            set { _idAlquiler = value; }
        }

        private List<AlquilerDTO> _listaAlquiler;

        public List<AlquilerDTO> listaAlquiler
        {
            get { return _listaAlquiler; }
            set { _listaAlquiler = value; }
        }

        private ErrorBean _error;

        public ErrorBean error
        {
            get { return _error; }
            set { _error = value; }
        }

    }
}
